### DISCUSSION TITLE
where to file bug report for gcc 11.2 brew?

### DISCUSSION THREAD

**Discussion by victor-tsang** [Homebrew/discussions#3044] [Tap maintenance and Homebrew development]
I experienced a fault situation when found out to be a fault of g++ 11.2 compiler. I filed a bug report to gcc and they closed quickly  with a response saying that aarch64 darwin (apple M1) is not supported... They also reminded me the g++-11 -v printed " --with-bugurl=https://github.com/Homebrew/homebrew-core/issues" and I should file the bug there.

However, the link https://github.com/Homebrew/homebrew-core/issues looks like a place for homebrew itself...

Where can I get the relevant people aware about the gcc bug?

```
 % g++-11 -v
Using built-in specs.
COLLECT_GCC=g++-11
COLLECT_LTO_WRAPPER=/opt/homebrew/Cellar/gcc/11.2.0_3/libexec/gcc/aarch64-apple-darwin21/11/lto-wrapper
Target: aarch64-apple-darwin21
Configured with: ../configure --prefix=/opt/homebrew/Cellar/gcc/11.2.0_3 --libdir=/opt/homebrew/Cellar/gcc/11.2.0_3/lib/gcc/11 --disable-nls --enable-checking=release --with-gcc-major-version-only --enable-languages=c,c++,objc,obj-c++,fortran --program-suffix=-11 --with-gmp=/opt/homebrew/opt/gmp --with-mpfr=/opt/homebrew/opt/mpfr --with-mpc=/opt/homebrew/opt/libmpc --with-isl=/opt/homebrew/opt/isl --with-zstd=/opt/homebrew/opt/zstd --with-pkgversion='Homebrew GCC 11.2.0_3' --with-bugurl=https://github.com/Homebrew/homebrew-core/issues --build=aarch64-apple-darwin21 --with-system-zlib --disable-multilib --with-native-system-header-dir=/usr/include --with-sysroot=/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk
Thread model: posix
Supported LTO compression algorithms: zlib zstd
gcc version 11.2.0 (Homebrew GCC 11.2.0_3) 
```

**Comment by SMillerDev** [comment]
https://github.com/fxcoudert/gcc is the fork we use for ARM support.

**Comment by carlocab** [comment]
https://github.com/Homebrew/homebrew-core/issues is for formulae-related issues. https://github.com/Homebrew/brew/issues is for `brew` itself.
